src/transformers/modeling_flax_utils.py

```diff
@@ -722,7 +770,13 @@ def from_pretrained(
                 filename = pretrained_model_name_or_path
                 resolved_archive_file = download_url(pretrained_model_name_or_path)
             else:
-                filename = WEIGHTS_NAME if from_pt else FLAX_WEIGHTS_NAME
+                if from_pt:
```

Comment:
If we set `from_pt=True` but safetensors weights are available, then we still use the safetensors weights no? So should we re-order this if/else logic to:
```python
                if is_safetensors_available():
                    filename = SAFE_WEIGHTS_NAME
                elif from_pt:
                    filename = WEIGHTS_NAME
                else:
                    filename = FLAX_WEIGHTS_NAME
```
This then matches the order we set when we check the model path

Reply:
Correct!

Addressed in https://github.com/huggingface/transformers/pull/27064/commits/1bd29e03c09284653e1d8ae4a8ed426299b99083

Reply:
Actually thinking more about it, I think that when using `from_pt=True` we'll want to continue forcing the load of the `pytorch_model.bin`. The safetensors file will be used as default in all other cases